packages/shared/src/organisms/TestnetAlert/TestnetAlert.tsx

```diff
@@ -0,0 +1,35 @@
+import { Alert, AlertTitle, useTheme } from '@mui/material';
+import { Trans, useTranslation } from 'react-i18next';
+
+interface TestnetAlertProps {
+  url: string;
+}
+
```

Comment:
Initiate const `PRODUCTION_URL` in this file

Reply:
Done